training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Mlflow PythonModel wrapper class that loads the Mlflow model, preprocess inputs and performs inference."""
```

Comment:
Are all files under vision used for prediction or part of scoring script?

Reply:
Discussed offline, yes, these all are part of prediction script.